HCCGo/app/js/dataUsageService.js

```diff
@@ -0,0 +1,42 @@
+dataUsageService = angular.module('dataUsageService', []);
+
+/**
+ * The dataUsageService allows for faster loading of data grabbed from the cluster interface.
+ * The service is called periodically, allowing for non-constant loading of statistics.
+ *
+ * A Promise with the data is returned, which is rendered within the controller.
+ *
+ * @ngdoc service
+ * @memberof HCCGo
+ * @class dataUsageService
+ */
+dataUsageService.service('dataUsageService',['$q', '$log', function($q, $log) {
+    var oldData = null;
+    var lastRequestedTime = 0;
+	return {
+
+	   /**
+	 	 * Gets storage usage from cluster
+		 * @method getDataUsage
+		 * @memberof HCCGo.dataUsageService
```

Comment:
Could you add a `@param` for the clusterInterface and force arguments:
http://usejsdoc.org/tags-param.html